config/commands.js

```diff
@@ -385,7 +385,7 @@ var commands = exports.commands = {
 				if (move.flags['sound']) details["<font color=black>&#10003; Sound</font>"] = "";
 				if (move.flags['bullet']) details["<font color=black>&#10003; Bullet</font>"] = "";
 				if (move.flags['pulse']) details["<font color=black>&#10003; Pulse</font>"] = "";
-				if (move.flags['protect']) details["<font color=black>&#10003; Blocked by Protect</font>"] = "";
+				if (!move.flags['protect'] && !/(ally|self)/i.test(move.target)) details["<font color=black>&#10003; Ignores Protect</font>"] = "";
```

Comment:
Only the 4 hazard moves are in the "foeSide" category and you can't block them with protect. That might seem like common sense, but I believe most people see hazards as targeting the opponent and having all moves that target the opponent be ones that can be listed as "Ignores Protect" seems the most clear.
"all" is much trickier. I understand what you're saying with the weather and terrain moves, which make up a majority of the moves int he "all" category; however some moves like Haze and Perish Song seem useful to list as ignoring protect because that's not common knowledge. I'm not sure if writing exceptions for specific moves is a good idea, and even though saying weather moves ignore protect is rather silly and useless I don't think it's particularly harmful.